src/navigation/recovery.py

```diff
@@ -54,7 +53,9 @@ def evaluate(self, ud) -> str:
                 dir_vector = -1 * RECOVERY_DISTANCE * pose.rotation.direction_vector()
                 self.waypoint_behind = pose.position + dir_vector
 
-            cmd_vel, arrived_back = get_drive_command(self.waypoint_behind, pose, STOP_THRESH, DRIVE_FWD_THRESH, True)
+            cmd_vel, arrived_back = self.context.rover.get_drive_command(
```

Comment:
I think we forgot to change this to match your new class, but this should be self.context.rover.driver.get_drive_command. Same for line 74. And we should maybe add "in_odom=self.context.use_odom" instead of the True here?

Reply:
yea, I fixed the call but I'm going to leave the driving back as is because your guys' PR will take care of it